internal/pkg/levee/levee.go

```diff
@@ -79,16 +102,114 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+func doPointerAnalysis(pass *analysis.Pass, analysisConf *config.Config, c *ssa.Call) (analyzedSuccesfully bool) {
+	// Pointer analysis requires at least one main package containing a main function.
+	// If we fail to provide such a package, the analysis will fail and produce error output in stderr.
+	pkg := c.Parent().Pkg
+	main := pkg.Func("main")
```

Comment:
Mentioned offline / in my above top-level comment, but I'm concerned about how this interacts with the `analysis` partition into package `pass`es.  Add tests that cross package boundaries.

Reply:
Please see my comment on the PR itself for further discussion.